experimental/dds/attributable-map/src/mapKernel.ts

```diff
@@ -675,7 +675,7 @@ export class AttributableMapKernel {
 		local: boolean,
 		localOpMetadata: MapLocalOpMetadata,
 	): boolean {
-		const op: IMapKeyOperation = message.contents;
+		const op = message.contents as IMapKeyOperation;
```

Comment:
@Abe27342 @clarenceli-msft in the original mapKernel, this cast isn't needed because we're taking an IMapKeyOperation directly.  Maybe consider passing the information you need for attribution as an additional param rather than relaxing the typing of the op coming through.